pkgs/development/python-modules/pyinfra/default.nix

```diff
@@ -30,6 +31,15 @@ buildPythonPackage rec {
     hash = "sha256-BYd2UYQJD/HsmpnlQjZvjfg17ShPuA3j4rtv6fTQK/A=";
   };
 
+  patches = [
+    # https://github.com/Fizzadar/pyinfra/pull/1018
+    (fetchpatch {
+      name = "bump-paramiko-major-version.patch";
+      url = "https://github.com/Fizzadar/pyinfra/commit/62a8f081279779c4f1eed246139f615cf5fed642.patch";
+      hash = "sha256-aT9SeSqXOD76LFzf6R/MWTtavcW6fZT7chkVg9aXiBg=";
+    })
+  ];
```

Comment:
Dp we really need a patch for that? shouldn't `pythonRelaxDepsHook` fix this in our case?

Reply:
It would work, and r-ryantm would then be more often able to make bump PRs, but I still prefer patches since they either cherry pick commits merged into the upstream, or at least link to an upstream discussion. If upstream later determines the dep relaxation breaks their app then i, who often watch such issues, can quickly make a PR to nixpkgs addressing this.

Reply:
> can quickly make a PR to nixpkgs addressing this

don't have to, just wondered ^^

Reply:
Ah, I meant if the upstream issue discovers a problem, then I can open a pr here